README.md

```diff
@@ -82,19 +83,21 @@ When deploying the multi cluster traffic controller using the make targets the f
     make deploy-agent
     ```
     
-## 4. Run the ingress agent locally
+## 4. Running the agent locally
 1. Target the workload cluster you wish to run on:
 ```sh
-export KUBECONFIG=./hack/kubeconfigs/mctc-workload-1.kubeconfig
+kubectl config use-context kind-mctc-workload-1
 ```
-1. Create the control plane secret
-Update the `samples/syncer/secret.yaml` from the `hack/kubeconfigs/mctc-control-plane.kubeconfig` and apply it:
+1. Update the secret in `config/agent/secret.yaml` with the correct credentials for the control plane `tmp/kubeconfigs/mctc-control-plane.kubeconfig` . (**N.B.** The server should be chnaged to the kind cluster address e.g. ``)
+
+1. Apply the control plane secret
 ```sh
-kubectl create -f samples/syncer/secret.yaml
+kubectl create ns mctc-system
+kubectl create -f config/agent/secret.yaml
 ```
 1. Run the agent locally:
 ```sh
-make run-agent
+make build-agent run-agent
```

Comment:
There are actually a few other things required to make all this work locally, but I'm not doing that in this PR.

Reply:
Related https://github.com/Kuadrant/multi-cluster-traffic-controller/pull/86